server/services/conditionsService.ts

```diff
@@ -45,7 +46,7 @@ export class ConditionsService {
 
   // form generation
   getFullTextForApprovedConditions(licence: Licence) {
-    const standardConditionsText = standardConditions.map((it) => it.text.replace(/\.+$/, ''))
+    const standardConditionsText = standardConditionsV1.map((it) => it.text.replace(/\.+$/, ''))
```

Comment:
Bit of blurred lines here between the tickets. I thought I'd not jump down the rabbit hole of updating this line to dynamically get the versions, as it does feel like there will be a fair few changes needed to do it consistently with the current approach. Plus, the other changes implemented are user-facing, but this presumably does open the door to visual inconsistency.

That being said, I could add a parameter to the function for the version and use the `getStandardConditionsByVersion` method.

Happy either way 😄

Reply:
Scratch what I said above; the int tests weren't going to work without updating it 😂 

I've now added the parameter in the class. I've added a unit test around this which is a bit narly with how it sits with other tests.

The following ticket will encompass refactoring this area, though, so we can better handle different versions of additional and standard conditions.